demos/database table/workflow_database_tables.sql

```diff
@@ -0,0 +1,46 @@
+SET sql_mode = STRICT_ALL_TABLES;
+
+DROP TABLE IF EXISTS Workflow;
+DROP TABLE IF EXISTS Version;
+DROP TABLE IF EXISTS Runs;
+DROP TABLE IF EXISTS Workflow_Data;
+DROP TABLE IF EXISTS Users;
+
+
+CREATE TABLE Workflow(
+	workflow_id INT NOT NULL AUTO_INCREMENT,
+	PRIMARY KEY(workflow_id)
+) ENGINE = InnoDB;
+	
+CREATE TABLE Version(
+	version_num INT NOT NULL, 
+	workflow_id INT NOT NULL,
+	PRIMARY KEY(workflow_id),
+	FOREIGN KEY (workflow_id) REFERENCES Workflow(workflow_id)
+) ENGINE = InnoDB;
+
+CREATE TABLE Runs(
+	version_num INT NOT NULL,
+	run_num INT NOT NULL,
+	graph_output LONGBLOB NOT NULL,

```

Comment:
i'm still not sure if this should be associated with version or run, might need to discuss